### ISSUE TITLE
Cannot edit existing geozone

### ISSUE THREAD

**Issue by bizkit1** [vitalidze/traccar-web#751]
I cannot modify the position/size of an existing geozone. I can see the dots from where i can click and drag, but it doesn't stretch or anything. Any ideea why?
Thanks!

**Comment by vitalidze** [comment]
I think there can be some overlapped layer, which prevents you from editing. Try to disable 'seamark' layer for example. This can be a bug too, if you can reproduce it on the demo site then please let me know your user account and name of geo-fence.

**Comment by bizkit1** [comment]
Yes i was able to reproduce it on the demo site. It works only if i disable the "Markers" Overlayer.
